pydra/engine/tests/test_task.py

```diff
@@ -171,7 +162,7 @@ def test_shell_cmd(tmpdir):
     assert res.output.stdout == " ".join(cmd[1:]) + "\n"
 
     # separate command into exec + args
-    shelly = ShellCommandTask(executable=cmd[0], args=cmd[1:])
+    shelly = ShellCommandTask(name="test", executable=cmd[0], args=cmd[1:])
```

Comment:
should not be necessary.

Reply:
was needed because `ShellCommandTask` didn't have default value for `name`. Added None